webapp/components/wpt-amend-metadata.js

```diff
@@ -286,6 +287,10 @@ class AmendMetadata extends LoadingState(PathInfo(ProductInfo(PolymerElement)))
       return `https://bugs.chromium.org/p/chromium/issues/list?q="${testName}"`;
     }
 
+    if (product === 'deno') {
+      return `https://github.com/denoland/deno/issues?q="${testName}"`;
```

Comment:
Cool, even supporting the newer features of wpt.fyi :)

Note that if you try to use the triage tooling for Deno, some changes to https://github.com/web-platform-tests/wpt-metadata might turn out to also be required.